### ISSUE TITLE
ICS export does not export all events.

### ISSUE THREAD

**Issue by matclab** [nextcloud/calendar#5938]
### Steps to reproduce

1. Create a single ponctual event today with a specific name (XXXXXXKKKKKK)
2. Get the ICS from public link (something like `https://calendar.example.com/remote.php/dav/public-calendars/XXXXXXXXXXX?export`)
3. Search XXXXXXKKKKKK in the obtained ICS file

### Expected behavior

The string XXXXXKKKKK would appear in the ICS file

### Actual behaviour

The string XXXXXKKKKK does not appear in the ICS file

### Calendar app version

4.6.7

### CalDAV-clients used

none (simple curl)

### Browser

curl

### Client operating system

Arch linux

### Server operating system

Manages nextcloud on Hetzner

### Web server

None

### Database engine version

None

### PHP engine version

None

### Nextcloud version

28.0.3

### Updated from an older installed version or fresh install

Updated from an older version

**Comment by SebastianKrupinski** [comment]
Hi @matclab 

I could not replicate your behavior, my tests produced the correct output. Are you sure you created the event in the correct calendar?

**Comment by matclab** [comment]
In fact, I went to testing this scenario after observing that some events I just added were not reported by the script [i3-agenda](https://github.com/rosenpin/i3-agenda) I use. At first, I was thinking it was because of the synchronization delay between nextcloud and the google agenda I poll, but then I observed that the event didn't appear in the ICS (whereas older repeating events appears).

But, I just tried again, and I'm unable to reproduce.  Either it was just a punctual problem on my instance or I make a mistake while testing.

I close the issue, and will reopen if the problem appears again.

Thank you very much for your time trying to reproduce it.
